packages/main/src/Popup.js

```diff
@@ -422,6 +438,11 @@ class Popup extends UI5Element {
 
 		this._lastFocusableElement = null;
 	}
+
+	disconnectedCallback() {
```

Comment:
we had a discussion - we could use this approach and remove the onEnterDOM, onExitDOM lifecycle methods, but then users would have to know that onEnterDOM is async and await the super call which is more error-prone than the lifecycle method, and for consistency, onExitDOM shloud also stay. 

For this case, please switch to onExitDOM (this will also spare you the super call).